systemd/fleet.service

```diff
@@ -4,7 +4,7 @@ After=network.target
 Requires=network.target
 
 [Service]
-ExecStart=/usr/bin/fleet -c /usr/share/fleet/fleet-server.yml
+ExecStart=/usr/bin/fleet-server -c /usr/share/fleet/fleet-server.yml
```

Comment:
I assume this would have to change if https://github.com/elastic/fleet-server/pull/57 is merged?

Reply:
yep, will adjust again, if we are changing the binary name once that PR landed